legacy/builder/gcc_preproc_runner.go

```diff
@@ -61,7 +61,8 @@ func prepareGCCPreprocRecipeProperties(ctx *types.Context, sourceFilePath *paths
 	properties.SetPath(constants.BUILD_PROPERTIES_SOURCE_FILE, sourceFilePath)
 	properties.SetPath(constants.BUILD_PROPERTIES_PREPROCESSED_FILE_PATH, targetFilePath)
 
-	includesStrings := utils.Map(includes.AsStrings(), utils.WrapWithHyphenI)
+    ctx.SetGlobalIncludeOption()
```

Comment:
Please check the spacing throughout the change.

Reply:
All 4-spaces are now replaced by tabs.